### ISSUE TITLE
install should not fail silently

### ISSUE THREAD

**Issue by dylansmith** [imagemin/optipng-bin#94]
We've had an issue today where our CI build failed silently because optipng-bin failed to install as a result of a 504:

```
> optipng-bin@3.1.4 postinstall /src/node_modules/optipng-bin
> node lib/install.js

  ⚠ spawn /src/node_modules/optipng-bin/vendor/optipng ENOENT
  ⚠ optipng pre-build test failed
  ℹ compiling from source
  ✖ HTTPError: Response code 504 (Gateway Time-out)
    at EventEmitter.<anonymous> (/src/node_modules/got/index.js:197:24)
    at emitOne (events.js:96:13)
    at EventEmitter.emit (events.js:188:7)
    at Immediate.<anonymous> (/src/node_modules/got/index.js:63:8)
    at runCallback (timers.js:672:20)
    at tryOnImmediate (timers.js:645:5)
    at processImmediate [as _immediateCallback] (timers.js:617:5)
```

As a result, our service was stuck in a death loop. It would be far preferable if `install.js` did a `process.exit(1)` or re-throw rather than simply `return`.

**Comment by aywan** [comment]
Same problem. 
Sourceforge.net ip resolved in 216.105.38.10, and this ip refuse connection.
```
Connecting to downloads.sourceforge.net (downloads.sourceforge.net)|216.105.38.10|:443... failed: Connection refused.
```
Spend 4 hour to find why it not work, and then why on another PC version of optipng 0.7.7, while on my laptop 0.7.5

**Comment by sindresorhus** [comment]
Fixed by https://github.com/imagemin/optipng-bin/commit/6bb7f1c2edab67297c7d72fa0bd604ecc214b9f1
